app/services/arcgis_api/token_keeper.rb

```diff
@@ -0,0 +1,247 @@
+module ArcgisApi
+  # Struct to store token information, this allows us to track
+  # real expiration time with various rails cache backends many of them
+  # do not support entry expiration.
+  # @member token the token string
```

Comment:
what does @member signify?

Reply:
yard doc tag, i will remove it since not used in this project
```ruby
module YARD::Handlers::Ruby::StructHandlerMethods
  # Extracts the user's defined @member tag for a given class and its member. Returns
  # nil if the user did not define a @member tag for this struct entry.
```